docs/TESTS.md

```diff
@@ -1,44 +1,110 @@
-### Creating the project in Xcode
+## Xcode Tutorial
 
-Please see <a href="https://developer.apple.com/library/tvos/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/02-quick_start.html#//apple_ref/doc/uid/TP40014132-CH2-SW1" target="_blank">Apple's guide on XCTest</a>
+This guide explains how to create and configure an Xcode project for Exercism.
 
-These steps will use the `bob` exercise as an example. If you are not working on `bob`, replace `bob` with your exercise name in the steps below.
+These instructions are written with the expectation that some readers will be very new to Xcode. Each step is described in excruciating detail. However, with a little experience performing these instructions should only take a minute.
 
-* Select iOS-->Application and then Single View Application and click Next.
+_As far as housekeeping chores go, you are closer to the finish line than it appears._
 
-* Enter `bob-swift`* as your Product Name, set the language to Swift, and check "Include Unit Tests", and then click Next.
+### Create A New Xcode Project
+![001-splash](https://cloud.githubusercontent.com/assets/5385825/20957611/c756745a-bc05-11e6-8631-e9e738ad464b.png)
 
-* Save the project in the same folder as the Exercism test file:
+Select macOS and Cocoa Application. (iOS Single View Application can also be used, but choosing macOS avoids the time needed to launch the iOS simulator each time a test is run.)
```

Comment:
This could use a little clarification. It could sound as if one is selecting a 'macOS and Cocoa Application', one step rather than two. I’d suggest changing to 'Select the macOS tab, then Cocoa Application'.

For the sake of simplicity, I’d consider dropping this parenthetical aside (iOS Single View Application, etc.), since it’s not much help to users anxious just to get started.